Formatters/GitHubCodespaces.Format.ps1xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Configuration>
+  <ViewDefinitions>
+    <!--================== GitHub.Codespace Type View =================-->
+    <View>
+      <Name>GitHub.Codespace</Name>
+      <ViewSelectedBy>
+        <TypeName>GitHub.Codespace</TypeName>
+      </ViewSelectedBy>
+      <TableControl>
+        <TableRowEntries>
+          <TableRowEntry>
+            <TableColumnItems>
+              <TableColumnItem>
+                <PropertyName>id</PropertyName>
+              </TableColumnItem>
+              <TableColumnItem>
+                <PropertyName>name</PropertyName>
+              </TableColumnItem>
+              <TableColumnItem>
+                <PropertyName>display_name</PropertyName>
+              </TableColumnItem>
+              <TableColumnItem>
+                <PropertyName>state</PropertyName>
+              </TableColumnItem>
+              <TableColumnItem>
+                <PropertyName>geo</PropertyName>
```

Comment:
If `geo` isn't in the response data, we probably don't want to have this in our formatter as this column then will always be blank.